src/pages/walled-gardens.md

```diff
@@ -3,17 +3,162 @@ title: 'Bringing Competition to Walled Gardens'
 permalink: '/walled-gardens-report/'
 layout: 'layouts/paper.njk'
 metaDesc: 'The full Bringing Competition to Walled Gardens report, published by Open Web Advocacy.'
-subtitle: 'Third Party Browsers & Web Apps - VERSION 1.0'
-paperName: 'OWA - Bringing Competition to Walled Gardens - v1.0'
+subtitle: 'Third-Party Browsers & Web Apps - VERSION 1.2'
```

Comment:
Ah I did a search-and-replace on that because we were using both the with and without hyphen version.

Will double check with the team on what's correct.